templates/admin-body.html

```diff
@@ -84,7 +84,7 @@ <h4 id="progress-title">Progress ({{ count($sites) }})
         <div class="col-md-4 col-xs-12" style="overflow-x: hidden; overflow-y: scroll; height: 440px;">            
             <ul id="sites">
                 {% foreach ($sites as $site) { %}
-                <li data-state="{{ $site['state'] }}">
+                <li data-state="{{ $site['state'] }}" data-title="{{ $site['title'] }}">
```

Comment:
If you add `data-siteid="{{ $site['site_id'] }}"` to the li you could also change the search to use the site id - very useful.

Reply:
And then add to check